robonomics/frame/datalog/src/weights.rs

```diff
@@ -0,0 +1,21 @@
+use frame_support::weights::{constants::RocksDbWeight as DbWeight, Weight};
+
+pub trait WeightInfo {
+    fn record() -> Weight;
+    fn erase(win: u64) -> Weight;
+}
+
+#[allow(clippy::unnecessary_cast)]
+impl WeightInfo for () {
+    fn record() -> Weight {
+        (1_000_000_u64 as Weight)
```

Comment:
How this constant was estimated? Let's generate it using benchmarks.

Reply:
This  weight constant was was received by running datalog benchmarks . Though, this isn't exact  value  since it assumes using specific hardware as described  in manual https://www.shawntabrizi.com/substrate-graph-benchmarks/docs/#/assumptions?id=processor